specs/eventing/test-plan/error-signaling-conformance.md

```diff
@@ -0,0 +1,470 @@
+# Error Signaling
+
+#### From:
+
+https://github.com/knative/specs/blob/main/specs/common/error-signalling.md
+
+#### From the Spec:
+
+> [...] Each user-visible resource described in [Resource
+> Overview][res-overview] MUST have a `conditions` field in `status`, which MUST
+> be a list of `Condition` objects described by the following table.
+>
+> Fields in the condition which are not marked as "REQUIRED" MAY be omitted to
+> indicate the default value [...]. As `Conditions` are an output API, an
+> implementation MAY never set these fields; the OpenAPI document MUST still
+> describe these fields.
+
+> Additionally, the resource's `status.conditions` field MUST be managed as
+> follows to enable clients (particularly user interfaces) to present useful
+> diagnostic and error message to the user. In the following section, conditions
+> are referred to by their `type` (aka the string value of the `type` field on
+> the Condition).
+>
+> 1.  Each resource MUST have a summary `Ready` condition (for ongoing systems)
+>     or `Succeeded` condition (for resources that run to completion) with
+>     `severity=""`, which MUST use the `"True"`, `"False"`, and `"Unknown"`
+>     status values as follows:
+>
+>     1.  `"False"` MUST indicate a failure condition.
+>
+>     [...]
+>
+> 1.  For non-`Ready` conditions, any conditions with `severity=""` (aka "Error
+>     conditions") MUST be aggregated into the "Ready" condition as follows:
+>
+>     1.  If the condition is `"False"`, `Ready`'s status MUST be `"False"`.
+>     1.  If the condition is `"Unknown"`, `Ready`'s status MUST be `"False"` or
+>         `"Unknown"`.
+>     1.  If the condition is `"True"`, `Ready`'s status can be any of `"True"`,
+>         `"False"`, or `"Unknown"`.
+>
+>     Implementations MAY choose to report that `Ready` is `"False"` or
+>     `"Unknown"` even if all Error conditions report a status of `"True"` (i.e.
+>     there might be additional hidden implementation conditions which feed into
+>     the `Ready` condition which are not reported.)
+>
+> 1.  Conditions with a `status` other than `"True"` SHOULD provide `message`
+>     and `reason` fields indicating the reason that the `status` is not
+>     `"True"`. Conditions where the `status` is `"False"` MUST provide a
+>     failure `reason` in the condition. (`"Unknown"` conditions might not have
+>     been reconciled, and so MAY have an empty `reason`.)
+
+[res-overview]: https://github.com/knative/specs/blob/c348f501/specs/eventing/overview.md
+
+# Testing Error Signaling Conformance:
```

Comment:
Hey @antoineco , I can read `signalling` in most of the text and in the original spec, but also a couple occurences of `signaling` in text and file name.
Are those intended?

Reply:
Oh no, those are definitely typos! Thanks for spotting them.
It's fixed: [18803d43..180e962d](https://github.com/knative/specs/compare/18803d43cbb2fdf902150b75033d4b711250b4a9..180e962d17c2337c06ab72cda07e0576b62aab0a)